docs/aws-region-enable.md

```diff
@@ -30,21 +30,30 @@ This document provides the information on how to address and resolve missing AWS
     We would also like to get AWS public access enabled for this region. Please use the following command:
         "aws ec2 disable-image-block-public-access --region `<missing-region>`"
 
-    Let's also adjust the quota to match the other regions :
-        "aws service-quotas request-service-quota-increase --region=`<missing-region>` --service-code=ec2 --quota-code=L-0E3CBAB9 --desired-value 1000.00"
+    Let's also adjust the quota to match the other regions if necessary. If the current quota is less than 1000, let's bump it to at least 1000 with the following request:
+        "aws service-quotas request-service-quota-increase --region=ap-southeast-5 --service-code=ec2 --quota-code=L-0E3CBAB9 --desired-value 1000.00"
+
+    The quota codes can be viewed using the below command:
+	"aws service-quotas --service-code ec2 list-service-quotas"
```

Comment:
Let's re-arrange things so that we first tell them to run the command to see the current value, and only then instruct them to make the request for increase if it is needed

Reply:
Thats right, I changed the order .